src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Apprentice/ApprenticeshipDetailsCsvModel.cs

```diff
@@ -57,7 +59,8 @@ public class ApprenticeshipDetailsCsvModel
                 TotalAgreedPrice = $"{model.TotalAgreedPrice.Value as object:n0}",
                 ApprenticeConfirmation = model.ConfirmationStatus.ToDisplayString(),
                 Status = model.ApprenticeshipStatus.GetDescription(),
-                Alerts = GenerateAlerts(model.Alerts)
+                Alerts = GenerateAlerts(model.Alerts),
+                DeliveryModel = ToDeliveryModelDescription(model.DeliveryModel)
```

Comment:
You should include the trailing comma so that future diffs don't have the problem this one has of highlighting an irrelevant line.